pkgs/applications/networking/instant-messengers/franz/default.nix

```diff
@@ -14,7 +14,7 @@ in stdenv.mkDerivation rec {
   # don't remove runtime deps
   dontPatchELF = true;
 
-  nativeBuildInputs = [ autoPatchelfHook makeWrapper dpkg ];
+  nativeBuildInputs = [ autoPatchelfHook wrapGAppsHook dpkg ];
```

Comment:
This could be reduced to just one wrapper by setting both `$PATH` and `gappsWrapperArgs` at the same time: https://github.com/NixOS/nixpkgs/pull/55057/files#diff-2eae46082af88960c0456c899221863aR63

Reply:
I don't really understand what happens under the hood. Can you please elaborate on which build inputs can be removed? Feel free to open a new PR. I will close this one in that case.

Reply:
`wrapGAppsHook` automatically creates a wrapper using [`wrapProgram`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/wrap-gapps-hook.sh#L55) and because we already use it in `postFixup` we get two wrappers.

So we don't need to remove it as an input but set `dontWrapGApps = true;` so we can wrap manually in `postFixup`.
Adding `"''${gappsWrapperArgs[@]}"` to the `wrapProgram` below should do the manual part as @Mic92 mentioned there.

Reply:
@worldofpeace @Mic92 Thanks for the suggestion. I've modified the code accordingly and tested on my nixos machine. It seems to be working.